lib/Graphs/PTACallGraph.cpp

```diff
@@ -56,6 +56,29 @@ void PTACallGraphEdge::addInDirectCallSite(const CallBlockNode* call) {
 }
 //@}
 
+bool PTACallGraphNode::isReachableFromFunction(const SVFFunction *fun) const {
```

Comment:
Would you be able to merge this code into`isReachableBetweenFunctions `? It seems redundant to make two functions.